store/uistate/features/okrplanning/okr/index.ts

```
@@ -215,12 +215,10 @@ export const useOKRStore = create<OKRState>()(
             removedWeight / remainingKeyResults.length,
           );
 
-          const redistributedKeyResults = remainingKeyResults.map(
-            (kr: any) => ({
-              ...kr,
-              weight: Number(kr.weight || 0) + weightPerRemaining,
-            }),
-          );
+          let redistributedKeyResults = remainingKeyResults.map((kr: any) => ({
```

ESLint

[error] 'redistributedKeyResults' is never reassigned. Use 'const' instead.

(prefer-const)